ddp-workspace/projects/ddp-singular/src/app/components/pages/home/home.component.ts

```diff
@@ -13,4 +14,19 @@ export class HomeComponent {
   readonly BROAD_INSTITUTE_URL = 'https://www.broadinstitute.org';
   readonly BOSTON_HOSPITAL_URL = 'https://www.childrenshospital.org';
   readonly GENOME_MEDICAL_URL = 'https://www.genomemedical.com';
+  readonly CONSENT_PDF_URL = 'https://storage.googleapis.com/singular-dev-assets/consent_self.pdf';
+  readonly SALIVA_KIT_PDF_URL = 'https://storage.googleapis.com/singular-dev-assets/saliva_kit.pdf'
+  readonly SALIVA_KIT_PDF_NAME = 'Preview Kit Instructions';
+  readonly CONSENT_PDF_NAME = 'Preview Consent Form';
+  readonly PRESCREEN_BUTTON_LABEL = 'Sign me up!';
+
+
+  constructor(private analytics: AnalyticsEventsService){};
+
+  public clickPreScreening(): void {
+    this.analytics.emitCustomGtagEvent("sign_me_up_button_link", this.PRESCREEN_BUTTON_LABEL);
+  }
+  public clickDownload(clickText: string, clickUrl: string): void {
+    this.analytics.emitCustomGtagEvent("pdf_download", clickText, clickUrl);
+  }
```

Comment:
These custom event names (`sign_me_up_button_link` and `pdf_download`) I just manufactured/invented based on the spreadsheet in the JIRA ticket https://broadinstitute.atlassian.net/browse/DDP-6224.
![Screen Shot 2022-05-11 at 12 16 18 PM](https://user-images.githubusercontent.com/29984380/167898257-7874fb8b-20c7-456a-8607-79fc71d2903b.png)
More needed to be added!